Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -755,8 +755,8 @@ static void OnRemovePlayerMessage(NetworkMessage netMsg)
         {
             if (netMsg.conn.playerController != null)
             {
-                netMsg.conn.RemovePlayerController();
                 Destroy(netMsg.conn.playerController.gameObject);
+                netMsg.conn.RemovePlayerController();
```

Comment:
is this issue in master?
have you tested this?

great find,  this is a flat out memory leak,  as gameobjects are left around and never destroyed.